scripts/gceworker.sh

```diff
@@ -12,6 +12,7 @@ cd "$(dirname "${0}")"
 
 case ${1-} in
     create)
+    set -x
```

Comment:
Not a fan of random `-x`s. I'd just keep or remove them all (right now favoring keep until the script is mature)